api/sendmail/services/sendmail.js

```diff
@@ -0,0 +1,25 @@
+const nodemailer = require('nodemailer');
+
+// Create reusable transporter object using SMTP transport.
+const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: 'kuikuigaocn@gmail.com',
+    pass: '123gaokuikui',
```

Comment:
I can recommend that you changed the password because this is on the internet now.